.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 language: go
 
 go:
-  - 1.8
+  - 1.9
```

Comment:
No reason to move to 1.9, might as well have both 1.8 and 1.9.

Other than that, thanks for doing this, @tombuildsstuff!

Reply:
@marstr good point - updated to include 1.8 too :)